windows/Storage/fileIntegration.js

```diff
@@ -50,33 +47,6 @@ export function addFilesPaths(paths){
 }
 
 /**
- * This method will easily download from localhsot an Object, and save the file
- * into a specific path (dest).
- *
- * ToDo: Work with directories. (This saves only a file, not a directory)
- */
-export function saveFileToPath(objectID, dest){
-  return new Promise((success, failure) =>{
-    let url = `http://localhost:8080/ipfs/${objectID}`
```

Comment:
This was just... I am so happy that I removed it!
![iu](https://cloud.githubusercontent.com/assets/33528/22752729/e8c85070-ee39-11e6-9709-23da7b4603cd.gif)